Typo in emitDeclarationError for declarations

**TypeScript Version:**  2.7.2

**Search Terms:** emitDeclarationOnly

**Code**

https://github.com/Microsoft/TypeScript/blob/fa4619c5c17a6a592245069f416b91e2ea40c36d/src/compiler/program.ts#L2202-L2205

```ts
error TS5052: Option 'emitDeclarationOnly' cannot be specified without specifying option 'declarations'.
```

**Expected behavior:**
Should say
error TS5052: Option 'emitDeclarationOnly' cannot be specified without specifying option 'declaration'.

**Actual behavior:**
Says 'declarations'

--- a/src/compiler/program.ts
+++ b/src/compiler/program.ts
@@ -2201,7 +2201,7 @@ namespace ts {
 
             if (options.emitDeclarationOnly) {
                 if (!options.declaration) {
-                    createDiagnosticForOptionName(Diagnostics.Option_0_cannot_be_specified_without_specifying_option_1, "emitDeclarationOnly", "declarations");
+                    createDiagnosticForOptionName(Diagnostics.Option_0_cannot_be_specified_without_specifying_option_1, "emitDeclarationOnly", "declaration");
                 }
 
                 if (options.noEmit) {

--- a/tests/baselines/reference/declFileEmitDeclarationOnlyError1.errors.txt
+++ b/tests/baselines/reference/declFileEmitDeclarationOnlyError1.errors.txt
@@ -1,7 +1,7 @@
-error TS5052: Option 'emitDeclarationOnly' cannot be specified without specifying option 'declarations'.
+error TS5052: Option 'emitDeclarationOnly' cannot be specified without specifying option 'declaration'.
 
 
-!!! error TS5052: Option 'emitDeclarationOnly' cannot be specified without specifying option 'declarations'.
+!!! error TS5052: Option 'emitDeclarationOnly' cannot be specified without specifying option 'declaration'.
 ==== tests/cases/compiler/hello.ts (0 errors) ====
     var hello = "yo!";
     

--- a/tests/baselines/reference/declFileEmitDeclarationOnlyError2.errors.txt
+++ b/tests/baselines/reference/declFileEmitDeclarationOnlyError2.errors.txt
@@ -1,8 +1,8 @@
-error TS5052: Option 'emitDeclarationOnly' cannot be specified without specifying option 'declarations'.
+error TS5052: Option 'emitDeclarationOnly' cannot be specified without specifying option 'declaration'.
 error TS5053: Option 'emitDeclarationOnly' cannot be specified with option 'noEmit'.
 
 
-!!! error TS5052: Option 'emitDeclarationOnly' cannot be specified without specifying option 'declarations'.
+!!! error TS5052: Option 'emitDeclarationOnly' cannot be specified without specifying option 'declaration'.
 !!! error TS5053: Option 'emitDeclarationOnly' cannot be specified with option 'noEmit'.
 ==== tests/cases/compiler/hello.ts (0 errors) ====
     var hello = "yo!";

